src/vs/platform/editor/common/editor.ts

```diff
@@ -288,6 +288,15 @@ export interface IEditorOptions {
 	 * applied when opening the editor.
 	 */
 	viewState?: object;
+
+	/**
+	 * A transient editor will attempt to appear as preview and certain components
+	 * will decide to ignore the editor when it becomes active. This option is
```

Comment:
nitpick - it might be nice to say "(e.g. the history tracking)" after "certain components" just so that we know an example of what it would ignore.